hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM ghcr.io/mesosphere/mindthegap:v1.14.3
+ARG MINDTHEGAP_VERSION=v1.14.3
+
+FROM --platform=${BUILDPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERSION} as bundle_builder
```

Comment:
Making this into a multiplatform build saves a small amount of time by benefitting from layer caching (will only be run once for both amd64 and arm64 builds as the layers will be shared). Also doesn't require a new buildx builder instance.